mobilizon_reshare/cli/cli.py

```diff
@@ -67,22 +86,25 @@ def recap(settings_file):
     safe_execution(recap_main, settings_file=settings_file)
 
 
-@mobilizon_reshare.command(help="Print events in the database that are in STATUS")
+@mobilizon_reshare.command(help="List objects in the database with different criteria")
 @from_date_option
 @to_date_option
 @click.argument(
-    "status", type=click.Choice(list(status_name_to_enum.keys())),
+    "object",
+    type=click.Choice(list(status_name_to_enum.keys())),
```

Comment:
these should be two separate sub commands. It's bad practice to do it like this and will create problems later if we want to add options. Make a sub-command for events and one for publications